templates/actors/vehicle/sidebar.hbs

```diff
@@ -158,9 +158,17 @@
     {{/if}}
 
     {{!-- Speed --}}
+    {{#if showSpeed}}
     {{> "systems/dnd5e/templates/actors/parts/actor-trait-line.hbs" icon="fa-solid fa-shoe-prints"
         label="DND5E.Speed" configAction="movement" configLabel="DND5E.MOVEMENT.Action.Configure" unitsConfig=CONFIG.movementUnits
         data=(dnd5e-object value=system.attributes.movement.max units=system.attributes.movement.units) }}
+    {{/if}}
+    {{#if showTravel}}
```

Comment:
GoS ships show mph and mpd for speed so it would be nice to have an extra line here if the travel pace is calculated. But if messing around with units is too much of a pain, we can leave it for now.

Reply:
Maybe use `fa-sun` for now and I can tweak it later.